lib/models/clothing_size.dart

```diff
@@ -0,0 +1,23 @@
+class ClothingSize {
+  final _size;
+
+  ClothingSize.fromString(String size) : this._size = size;
+
+  ClothingSize.fromInt(int size) : this._size = size;
```

Comment:
I created these constructors. My idea is that we will replace these with .shoe(int size), .oneSize(), .regular(char size) etc when the design is ready.

Reply:
Good idea!